wagtail/snippets/wagtail_hooks.py

```diff
@@ -58,5 +58,6 @@ def register_snippet_listing_buttons(snippet, user, next_url=None):
             _('Delete'),
             reverse('wagtailsnippets:delete', args=[model._meta.app_label, model._meta.model_name, quote(snippet.pk)]),
             attrs={'aria-label': _("Delete '%(title)s'") % {'title': str(snippet)}},
-            priority=20
+            priority=20,
+            classes={'no'}
```

Comment:
Could we use a list instead of a set? This is for consistency, also I think that sets don't maintain ordering which would be important when using multiple classes